### ISSUE TITLE
append-5+ and prepend-5+ in grid.css have been removed from SVN trunk code.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [davidbelow/blueprintcss#17]
```
Hi,

In the latest version of the trunk code in SVN (rel 0.5 beta) you have
removed the append-5 and higher, and prepend-5 and higher css classes in
grid.css.  Now both append and prepend only go up to 4.

These were useful when for example I wanted to push a piece of text out to
column 14.  I could 'prepend-13 span-1' and it was done.

Now it seems I have to create a new empty column, with a span-13 followed
by my span-1 column.

Am am doing this in a way that is not proper?  Did I miss something in your
removal of these?

Thanks.


```

Original issue reported on code.google.com by `glenn.re...@gmail.com` on 16 Aug 2007 at 11:22

**Comment by GoogleCodeExporter** [comment]
```
I experimented to see if anybody actually used those. Thanks! :)
```

Original comment by `ola...@gmail.com` on 22 Aug 2007 at 12:03
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This is not fixed in v 0.5.  This version only has append 1-12 and prepend 1-12.

Without this, how can I have a 1 column wide menu item show in in grid position 
22
all the way over on the right side of the page for example?

Please limit experiments on those of us who are actualy using this tool.  We
suffer... :-)

Please re-open the issue.
```

Original comment by `glenn.re...@gmail.com` on 29 Aug 2007 at 3:15
